src/_utils.js

```diff
@@ -650,6 +658,10 @@ export const setStateOptions = state => {
       vendorPrefixes: state.opts.vendorPrefixes
     })
   }
+  state.styleModule =
+    typeof state.opts.styleModule === 'string'
+      ? state.opts.styleModule
+      : 'styled-jsx/style'
```

Comment:
we should document this in the readme

Reply:
We already have [this](https://github.com/vercel/styled-jsx#stylemodule), I just moved this part here from another place